src/libpgmoneta/prometheus.c

```diff
@@ -2195,6 +2195,89 @@ size_information(int client_fd)
       data = NULL;
    }
 
+   data = pgmoneta_append(data, "#HELP pgmoneta_backup_newest_throughput The throughput of the newest backup for a server\n");
+   data = pgmoneta_append(data, "#TYPE pgmoneta_backup_newest_throughput gauge\n");
+   for (int i = 0; i < config->number_of_servers; i++)
+   {
+      d = pgmoneta_get_server_backup(i);
+
+      number_of_backups = 0;
+      backups = NULL;
+
+      pgmoneta_get_backups(d, &number_of_backups, &backups);
+
+      if (number_of_backups > 0)
+      {
+         for (int j = 0; j < number_of_backups; j++)
+         {
+            if (backups[j] != NULL)
+            {
+               data = pgmoneta_append(data, "pgmoneta_backup_newest_throughput{");
+
+               data = pgmoneta_append(data, "name=\"");
+               data = pgmoneta_append(data, config->servers[i].name);
+               data = pgmoneta_append(data, "\",label=\"");
+               data = pgmoneta_append(data, backups[j]->label);
+               data = pgmoneta_append(data, "\"} ");
+
+               if (backups[j]->elapsed_time)
+               {
+                  unsigned long throughput = backups[j]->backup_size / backups[j]->elapsed_time;
+                  if (throughput < 1024)
+                  {
+                     data = pgmoneta_append_double(data, 1.0 * throughput);
+                     data = pgmoneta_append(data, " B/s");
```

Comment:
I don't think prometheus allows string in the metrics. You need to run an prometheus instance to verify your result. See https://prometheus.io/docs/prometheus/latest/getting_started/

Reply:
Just settle for a fixed unit and docmuent it.

Reply:
Or alternatively you can keep the unit as a parameter, that should be better.

Reply:
Yes, it doesn't permit it. I just made it bytes/seconds and will try to mention it somewhere.